modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -81,77 +81,90 @@ export interface TabsItemProps
   tabIndex?: number;
 }
 
-export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItemProps, 'gap'>>(
-  ({theme}) => ({
-    ...type.levels.subtext.large,
-    fontWeight: type.properties.fontWeights.medium,
+const tabItemStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.medium,
     border: 'none',
     backgroundColor: 'transparent',
     flex: '0 0 auto',
+    minWidth: system.space.zero,
     alignItems: 'center',
-    padding: `${space.xs} ${space.s}`,
-    height: 52,
-    boxSizing: 'border-box',
+    padding: `${system.space.x3} ${system.space.x4}`,
+    height: px2rem(52),
     cursor: 'pointer',
-    color: colors.licorice300,
+    color: system.color.fg.muted.default,
     position: 'relative',
-    borderRadius: `${borderRadius.m} ${borderRadius.m} 0px 0px`,
+    borderRadius: `${system.space.x1} ${system.space.x1} ${system.space.zero} ${system.space.zero}`,
     transition: 'background 150ms ease, color 150ms ease',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    [systemIconStencil.vars.color]: system.color.fg.muted.default,
 
-    '&:hover, &:focus-visible, &.hover, &.focus': {
-      backgroundColor: colors.soap200,
-      color: colors.blackPepper400,
+    '&:has(span)': {
```

Comment:
can this be a data-part?

Reply:
This is for this portion here: https://github.com/Workday/canvas-kit/blob/b2e77e342788ae17ef1da20f480e71bcc205f32f/modules/react/tabs/lib/TabsItem.tsx#L147-L157

Reply:
It's for setting that
```tsx
display: 'flex', gap: 'space.xxs'
```

Reply:
I may use a `data-part` for specificity

Reply:
Ah, `Tabs.Item.Text` renders an `EllipsisText`. So we can probably add a `data-part` to `EllipsisText`

Reply:
I see what the code is doing. If you go to the `Icons` story, the tab renders two spans. One for the icon and one for the text. If it see that, it changes the display properties. You can keep the span or add data-attributes to both of those sub elements, up to you.

Reply:
This works for me. Adding those `data-part`'s